strawberry/schema/config.py

```diff
@@ -13,6 +15,7 @@ class StrawberryConfig:
     default_resolver: Callable[[Any, str], object] = getattr
     relay_max_results: int = 100
     disable_field_suggestions: bool = False
+    info_class: type[Info] = field(default_factory=lambda: Info)
```

Comment:
Looking at this again, the default can just be the class -- the mutability trap doesn't affect this.